src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,71 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Support for several Java 8 API added.
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8 by default
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate c.g.g.validation
+
+### Compiler
+- Update JS reserved keywords to ES6.
+- Fix instanceof for primitives & string.
+- Make exceptions work across Java/JavaScript boundary
+- Fix some warnings for Closure integration.
+- Allow casting native arrays to Object[].
+- Use patched JDT to circumvent memory leak when compiling for Java 8.
+
+### JsInterop
+- Implement @JsOverlay with default methods in Java8.
+- Fix JAnalysisDecoratedType regarding JsInterop.
```

Comment:
That's an internal detail that won't talk to many people (I, for one, don't understand what this actually means)